frontend/src/concepts/pipelines/content/artifacts/ArtifactUriLink.tsx

```diff
@@ -1,46 +1,38 @@
 import React from 'react';
```

Comment:
Also in this file, Andrew said to change the button to use an href instead window.open

```tsx
 <Button
          variant="link"
          isInline
          icon={<ExternalLinkAltIcon />}
          iconPosition="end"
          // TODO: remove this style override after upgrading to PFv6
          style={{ display: 'inline-flex' }}
          onClick={() => {
            window.open(url);
          }}
        >
```